ysoserial/ysoserial.csproj

```diff
@@ -91,6 +91,9 @@
     <Prefer32Bit>true</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="AspNetTicketBridge, Version=1.6.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Hopefully after just having the `System.IdentityModel.Services` as the reference, we will not any changes for any other libraries here.